### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by vitornogueira**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -110,7 +110,7 @@
                 <div class="col-md-3">
                     <h4>What is MEAN.JS?</h4>
                     <p>
-                        MEAN.JS is a full-stack JavaScript solution that helps you build fast, robust and maintainble production web applications using MongoDB, Express, AngularJS, and Node.js.
+                        MEAN.JS is a full-stack JavaScript solution that helps you build fast, robust and maintainable production web applications using MongoDB, Express, AngularJS, and Node.js.
```
Comment (ilanbiala) [inline index.html]: Can you also add a comma after robust?

**Comment by jloveland** [comment]
any reason not to merge this? seems trivial..

**Comment by ilanbiala** [comment]
@jloveland just slipped through the cracks. @vitornogueira can you squash down to 1 commit and then I'll merge?

**Comment by vitornogueira** [comment]
@ilanbiala  done :+1:

**Pull request merged**